### ISSUE TITLE
Suggestions for Poketwo Anniversary Event

### ISSUE THREAD

**Issue by poketwouser** [poketwo/poketwo#407]
As we all know Poketwo birthday is coming, why can't we have a b'day event. But with redeems down it's a difficult task to host a massive event. But a small event would be good. A catch event like Naughty and Nice Box event would be perfect. But as it's a anniversary time and comes only once q year I would like to have it big. It would be good if deems were back. And you could launch Anniversary Wooloo like pokemon as last year along with a shadow pokemon. Or will be a massive surprise for all, GEN 8 moves and new battles update in Poketwo. Will have to wait watch. Let this year anniversary event be a tribute to creators like release a pokemon called Oliver Groudon and Dagger Mace Kyorge.

**Comment by poketwouser** [comment]
From your beloved 2 years user  @Jase Holder ✨#8191 756099906980020245. Hope you will also unban the same. Got banned twice for spam advertisement. I will never do it again

**Comment by WitherredAway** [comment]
@poketwouser please appeal for your ban at https://forms.poketwo.net/a/ban-appeal , this place is not meant for that!

**Comment by UnknownUser135** [comment]
> As we all know Poketwo birthday is coming, why can't we have a b'day event. But with redeems down it's a difficult task to host a massive event. But a small event would be good. A catch event like Naughty and Nice Box event would be perfect. But as it's a anniversary time and comes only once q year I would like to have it big. It would be good if deems were back. And you could launch Anniversary Wooloo like pokemon as last year along with a shadow pokemon. Or will be a massive surprise for all, GEN 8 moves and new battles update in Poketwo. Will have to wait watch. Let this year anniversary event be a tribute to creators like release a pokemon called Oliver Groudon and Dagger Mace Kyorge.

Yes I also agree. Please enable the redeems, it’s been down for more than a month.
